ctwrap/_version.py

```diff
@@ -1,9 +1,10 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Please remove the header - this is a module within a package, and doesn't need those.